addons/im_livechat/static/src/bugfix/bugfix.scss

```diff
@@ -4,3 +4,9 @@
 * stable versions of Odoo. Any style that is defined in this file should be isolated
 * in its own file in master.
 */
+
+.modal-open {
+    .o_livechat_button {
```

Comment:
Did you try :) ?
This can't work as this is only loaded in backend assets, this should be in website_livechat bugfix file (or add one in im_livechat that is loaded in frontend), I guess it might be useful in im_Livechat but lets set it in website_livechat as this is a fix for stable version, keep it to website

Reply:
my bad, I tried it before moving it in this file and not retest it after